EFAuditer/EFAuditServiceExtensions.cs

```diff
@@ -3,17 +3,28 @@
 using Audit.Core;
 using EFAuditer;
 using Microsoft.EntityFrameworkCore;
+using Microsoft.AspNetCore.Http;
 
 namespace Microsoft.Extensions.DependencyInjection
 {
-    public static class EFAuditServiceExtensions
+  public static class EFAuditServiceExtensions
     {
         public static void AddEFAuditer(this IServiceCollection services, string connectionString)
         {
+            services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();

```

Comment:
Would this be better being moved to startup and using DI to get it in here? Similarly to userService.

Not massively fussed if it's only intended to be used here only.

Reply:
Hadn't clocked that this was in the Audit project. This is obviously moot in retrospect.